src/malcolm/reducer/navigation.reducer.js

```diff
@@ -191,7 +191,7 @@ function updateNavTypes(state) {
   // find main attribute
   updatedState.mainAttribute = undefined;
   let mainAttributeNav;
-  if (navigationLists.length !== 1) {
+  if (navigationLists.length >= 1) {
```

Comment:
Bug was actually on this line (I didn't account for a zero length navigation list)